htmd/parameterization/cli.py

```diff
@@ -366,7 +334,6 @@ def printEnergies(m, filename):
                 f.close()
             except ValueError as e:
                 print("Not writing Amber FRCMOD: {}".format(str(e)))
-    sys.exit(0)
```

Comment:
Why removing the `sys.exit`?

Reply:
Why do you need it at the end of the function.

Reply:
The `main_parameterize()` function is the one that is executed by the command line. As such, we have a `sys.exit(0)` at the end of it

Reply:
Exit status is set to 0 implicitly, if the there is no exceptions raised.

Reply:
ok, it was there just to be safe